app/models/countries.rb

```diff
@@ -0,0 +1,247 @@
+# This list is taken from https://www.gov.uk/api/foreign-travel-advice.json
+# $ curl https://www.gov.uk/api/foreign-travel-advice.json | jq '.details.countries[].name'
+
+module Countries
+  def self.all
```

Comment:
It's not clear to me why we want the constant to be private?

Why wouldn't I want to do something like this?

```rb

UK = 'United Kingdom'
ALL = ['Afghanistan', 'Albania', UK, 'Zimbabwe']

```